rpcserver.go

```diff
@@ -5104,8 +5105,17 @@ func median(s []dcrutil.Amount) dcrutil.Amount {
 		return 0
 	}
 
+	sort.Sort(dcrutil.AmountSorter(s));
+
 	middle := len(s) / 2
-	return s[middle]
+
+	if len(s) == 0 {
+		return 0
+	} else if (len(s) % 2) != 0 {
+		return s[middle]
+	} else {
```

Comment:
OK, if that would be the more idiomatic way. I'll make the change and go through with what @alexlyp mentioned.  Could have sworn I did a go fmt, but maybe that was with the dcrutil commit.